operations/helm/charts/mimir-distributed/templates/ingester/ingester-statefulset.yaml

```
@@ -186,6 +186,13 @@ spec:
           {{- $jaeger_queue_size := dig "jaegerReporterMaxQueueSize" nil .Values.ingester }}
           {{- if or .Values.global.extraEnv .Values.ingester.env $jaeger_queue_size }}
           env:
+            {{- $cpu_request := dig "requests" "cpu" nil .Values.ingester.resources }}
+            {{- if $cpu_request }}
+              {{- $cpu_request_between_3_and_6 := include "mimir.parseCPU" (dict "value" $cpu_request) | float64 | min 6 | max 3 }}
+            {{/* copy logic form jsonnet */}}
```

I wasn't sure which jsonnet it was until I looked at the other PR, this might make it clearer
```suggestion
            {{/* copy logic from operations/mimir/ingester.libsonnet */}}
```

---

good call, lots of jsonnet in our world